src/updaters/python/setup-py.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Update, UpdateOptions, VersionsMap } from '../update';
+import { GitHubFileContents } from '../../github';
+
+export class SetupPy implements Update {
+  path: string;
+  changelogEntry: string;
+  version: string;
+  versions?: VersionsMap;
+  packageName: string;
+  create: boolean;
+  contents?: GitHubFileContents;
+
+  constructor(options: UpdateOptions) {
+    this.create = false;
+    this.path = options.path;
+    this.changelogEntry = options.changelogEntry;
+    this.version = options.version;
+    this.packageName = options.packageName;
+  }
+  updateContent(content: string): string {
+    return content.replace(
+      /version ?= ?"[0-9]+\.[0-9]+\.[0-9](-\w+)?"/,
```

Comment:
Could this regex also allow for single quotes around the version?

Reply:
see no reason why not, although it would switch them to `"` when generating the new PR, is that a problem?

Reply:
It's fine if they get switched for our repositories. Most are using a code-formatter that will always use double-quotes, but we might have some stragglers.